src/celestia/qt/qtdeepskybrowser.cpp

```diff
@@ -3,7 +3,7 @@
 // Copyright (C) 2008, Celestia Development Team
 // celestia-developers@lists.sourceforge.net
 //
-// Deep sky browser widget for Qt4 front-end
+// Deep sky browser widget for Qt5-Qt6 front-end
```

Comment:
```suggestion
// Deep sky browser widget for Qt front-end
```
who knows when Qt7 arrives )